README.md

```diff
@@ -1,8 +1,18 @@
-# Newspack Google Ad Manager
+# Newspack Ad Manager
```

Comment:
```suggestion
# Newspack Ads
```

Shouldn't we preserve the plugin name at this title? We don't mention "Newspack Ad Manager" anywhere else.

Reply:
Yes! Nice catch - thank you!